.github/workflows/build-server.yml

```
@@ -0,0 +1,76 @@
+        with:
+          fetch-depth: 0
+
+      - name: Download artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: mt5server
+          path: dist
+
+      - name: Create Release
+        if: github.event.inputs.version == ''
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)